xmodule/seq_block.py

```diff
@@ -28,7 +28,6 @@
     HTMLSnippet,
     ResourceTemplates,
     shim_xmodule_js,
-    STUDENT_VIEW,
```

Comment:
Tiny nit: could we remove the one imported from `.x_module` instead? It seems to be a more commonly used pattern to import from `xmodule.x_module`.

Reply:
Yep, done, @Agrendalath.